test/metabase/driver/druid_test.clj

```diff
@@ -59,3 +62,123 @@
 (expect-with-engine :druid
   :completed
   (:status (process-native-query native-query-2)))
+
+
+;;; +------------------------------------------------------------------------------------------------------------------------+
```

Comment:
dumb question, but are these tests redundant now that there are tests in `metabase/query_processor_test/expression_aggregations_test.clj` ?